unstructured/partition/text_type.py

```diff
@@ -45,7 +46,13 @@ def is_possible_narrative_text(
         narrative text
     language
         The two letter language code for the text. defaults to "en" for English
+    language_checks
+        If True, conducts checks that are language to the specified language.
```

Comment:
```suggestion
        If True, conducts checks that are specific to the chosen language.
```
Also maybe add a note that selecting `True` could negatively impact performance?